python/ray/tune/schedulers/async_hyperband.py

```diff
@@ -204,6 +223,15 @@ def cutoff(self, recorded) -> Optional[Union[int, float, complex, np.ndarray]]:
     def on_result(self, trial: Trial, cur_iter: int, cur_rew: Optional[float]) -> str:
         action = TrialScheduler.CONTINUE
         for milestone, recorded in self._rungs:
+            if (
+                cur_iter >= milestone
+                and trial.trial_id in recorded
+                and not self._stop_last_trials
+            ):
+                # If our result has been recorded for this trial already, the
+                # decision to continue has already been made. Thus we can skip new
+                # cutoff calculation and just continue.
+                break
```

Comment:
Just double checking, should this be break or continue?

Reply:
Sounds good. Perhaps we can clarify here. "continue the trial"

Reply:
Updated the comment to clarify